### PR TITLE
Cesium3DTilesInspector attaches to first tileset on hover

### LINKED ISSUE #5139: Auto connect 3D Tiles Inspector to tileset
@austinEng @lilleyse what do you think?

Often we have one tileset loaded and then we bring up the inspector, but then have to pick on the tileset before we can inspect it.

Can we add a flag like `inspectFirstAvailableTileset` (better name?) that when `true` loops through `scene.primitives` and sets the inspector to the first `Cesium3DTileset` instance it finds?

Probably default to `false`.

CC #3241

### PR DESCRIPTION
**Pull request by ggetz**
Fixes #5139

`Cesium3DTilesInspector` will attach to the first tileset that is hovered over if no tileset is defined. If `tileset` is set, the behavior is disabled.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ Change Log
 * This is an npm-only release to fix an issue with using Cesium in Node.js.
 * Fixed a bug where Cesium would fail to load under Node.js and some webpack configurations. [#5593](https://github.com/AnalyticalGraphicsInc/cesium/issues/5593)
 * Fixed a bug where a Model's compressed textures were not being displayed. [#5596](https://github.com/AnalyticalGraphicsInc/cesium/pull/5596)
+* Fixed a bug where jsep was undefined when using webpack [#5593](https://github.com/AnalyticalGraphicsInc/cesium/issues/5593)
 * Fixed documentation for `OrthographicFrustum`. [#5586](https://github.com/AnalyticalGraphicsInc/cesium/issues/5586)
+* Added behavior to `Cesium3DTilesInspector` that selects the first tileset hovered over if no tilest is specified. [#5139](https://github.com/AnalyticalGraphicsInc/cesium/issues/5139)
```
Comment (mramato) [inline CHANGES.md]: This needs to go in a new 1.36 section.

File: CHANGES.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ Change Log
 * This is an npm-only release to fix an issue with using Cesium in Node.js.
 * Fixed a bug where Cesium would fail to load under Node.js and some webpack configurations. [#5593](https://github.com/AnalyticalGraphicsInc/cesium/issues/5593)
 * Fixed a bug where a Model's compressed textures were not being displayed. [#5596](https://github.com/AnalyticalGraphicsInc/cesium/pull/5596)
+* Fixed a bug where jsep was undefined when using webpack [#5593](https://github.com/AnalyticalGraphicsInc/cesium/issues/5593)
```
Comment (mramato) [inline CHANGES.md]: This can be removed, since it's a dupe of item 2 above.

**Comment by ggetz** [comment]
@mramato Updated.

Also do you know where this eslint error is coming from?
```
> cesium@1.35.2-master22602 eslint /home/travis/build/AnalyticalGraphicsInc/cesium
> eslint "./**/*.js" "./**/*.html" --cache --quiet
Unexpected token ...
SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/travis/build/AnalyticalGraphicsInc/cesium/node_modules/eslint-plugin-html/src/index.js:4:17)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

I'm at a loss.

**Comment by mramato** [comment]
> I'm at a loss.

Me too, I reran travis to see if it happens again or if it was some kind of fluke.

**Comment by mramato** [comment]
> Me too, I reran travis to see if it happens again or if it was some kind of fluke.

Fixed in #5617, it's affecting all branches.

**Comment by lilleyse** [comment]
Hover-over selection doesn't seem to work with these changes.

**Comment by ggetz** [comment]
Thanks @mramato ! 

@lilleyse I now restore the hover-over selection after the tileset is selected.

**Comment by lilleyse** [comment]
Looks good, can you remove the `.idea` included here?

**Comment by ggetz** [comment]
@lilleyse Removed

**Comment by lilleyse** [comment]
Thanks @ggetz

**Pull request merged**